labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,57 @@
+function startGame() {
+  const play = window.prompt("Do you want to play?");
+  if (play === "yes") {
```

Comment:
Using `play.toLowerCase()` here would allow the user to type "Yes" or "yes".

OR: A `confirm` might be nicer than a `prompt`.